tests/unit/plugins/kubernetes/kubernetes_pod_executor_test.py

```diff
@@ -116,6 +116,7 @@ def test_run(mock_get_node_affinity, k8s_executor):
         annotations={
             "paasta.yelp.com/some_annotation": "some_value",
         },
+        service_account_name="test_sa",
```

Comment:
Technically this isn't valid with the `_` right? Which I think means our regex match is only matching the first few valid chars here since this wasn't throwing any issues

Reply:
wow, can't believe this evaded us for so long